playwright-e2e/page-objects/local-authority/birth-father/laBirthFatherDocumentsPage.page.ts

```diff
@@ -0,0 +1,25 @@
+import { Locator, Page } from '@playwright/test';
+
+import BasePage from '../../../pages/basePage.page';
+
+export class LABirthFatherDocumentsPage extends BasePage {
```

Comment:
Class name confused me. Might be better to call it something like LABirthFatherServeDocumentsPage?

Reply:
100% agree, making the change to the file path and the class to reflect the upcoming change